package.json

```diff
@@ -5,10 +5,11 @@
   "type": "module",
   "main": "./dist/compound-web.js",
   "module": "./dist/compound-web.js",
+  "sideEffects": false,
```

Comment:
This tells downstream bundlers that there is no side-effect importing compound, which lets them actually treeshake stuff out